doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
+          kind: Config
+          clusters:
+          - cluster:
+              insecure-skip-tls-verify: true
+              server: http://${your_username}@localhost:8889
+            name: proxy-server
+          contexts:
+          - context:
+              cluster: proxy-server
+              user: admin/proxy-server
+            name: $namespace/proxy-server
+          current-context: $namespace/proxy-server
+          preferences: {}
+          users:
+          - name: admin/proxy-server
+            user:
+              username: ${your_username}
+              password: unused
+          ```
+    1. Encode the `Config` as base64:
+          ```sh
+          $ cat scorecard-config.yaml | base64 -w 0
+          YXBpVmVyc2lvbjogdjEKa2luZDogQ29uZmlnCmNsdXN0ZXJzOgotIGNsdXN0ZXI6CiAgICBpbnNlY3VyZS1za2lwLXRscy12ZXJpZnk6IHRydWUKICAgIHNlcnZlcjogaHR0cDovL2V5SmhjR2xXWlhKemFXOXVJam9pSWl3aWEybHVaQ0k2SWlJc0ltNWhiV1VpT2lKelkyOXlaV05oY21RaUxDSjFhV1FpT2lJaUxDSk9ZVzFsYzNCaFkyVWlPaUp2YkcwaWZRbz1AbG9jYWxob3N0Ojg4ODkKICBuYW1lOiBwcm94eS1zZXJ2ZXIKY29udGV4dHM6Ci0gY29udGV4dDoKICAgIGNsdXN0ZXI6IHByb3h5LXNlcnZlcgogICAgdXNlcjogYWRtaW4vcHJveHktc2VydmVyCiAgbmFtZTogL3Byb3h5LXNlcnZlcgpjdXJyZW50LWNvbnRleHQ6IC9wcm94eS1zZXJ2ZXIKcHJlZmVyZW5jZXM6IHt9CnVzZXJzOgotIG5hbWU6IGFkbWluL3Byb3h5LXNlcnZlcgogIHVzZXI6CiAgICB1c2VybmFtZTogZXlKaGNHbFdaWEp6YVc5dUlqb2lJaXdpYTJsdVpDSTZJaUlzSW01aGJXVWlPaUp6WTI5eVpXTmhjbVFpTENKMWFXUWlPaUlpTENKT1lXMWxjM0JoWTJVaU9pSnZiRzBpZlFvPQogICAgcGFzc3dvcmQ6IHVudXNlZAo=
+          ```
+    1. Create a `Secret` manifest `scorecard-secret.yaml` containing the operator's namespace (if any) the `Config`'s base64 encoding as a `spec.data` value under the key `kubeconfig`:
+          ```yaml
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: scorecard-kubeconfig
+            namespace: ${your_namespace}
+          data:
+            kubeconfig: ${kubeconfig_base64}
+          ```
+    1. Apply the secret in-cluster:
+          ```sh
+          $ kubectl apply -f scorecard-secret.yaml
+          ```
+    1. Insert a volume referring to the `Secret` into the operator's `Deployment`:
+          ```yaml
+          spec:
+            install:
+              spec:
+                deployments:
+                - name: memcached-operator
+                  spec:
+                    ...
+                    template:
+                      ...
+                      spec:
+                        containers:
+                        ...
+                        volumes:
+                        # scorecard kubeconfig volume
+                        - name: scorecard-kubeconfig
+                          secret:
+                            secretName: scorecard-kubeconfig
+                            items:
+                            - key: kubeconfig
+                              path: config
+          ```
+1. Insert a volume mount and `KUBECONFIG` environment variable into each container in your operator's `Deployment`:
```

Comment:
IMHO something as the following can help it be more clear to the users. 
```suggestion
    1. Insert a volume referring to the `Secret` into the CSV file in the install section like the following example. 
```
```suggestion
1. Insert a volume mount and `KUBECONFIG` environment variable into the CSV file in the install section like the following example.  
```

Reply:
Maybe should be better add all these in the operator.yml and ask to re-run the operator-sdk command to generated the CSV file. WDYT?